infra/growth/src/templates/emailInvite.txt

```diff
@@ -1,7 +1,5 @@
 ${referrerName} has invited you to join Origin.
 
```

Comment:
I think I had 2 newlines here on purpose. To force the content to show up in gmail with a newline. Otherwise gmail puts all the text all together, without any newline... Same comment for te reminder text email.